server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -180,8 +182,35 @@ export class DeliLambdaFactory extends EventEmitter implements IPartitionLambdaF
 					Lumberjack.info(message, getLumberBaseProperties(documentId, tenantId));
 				}
 			} else {
-				lastCheckpoint = JSON.parse(document.deli);
-			}
+                let checkpoint;
+                let isLocalCheckpoint = false;
+                if(this.localCheckpointEnabled){
+                    // Search local db for checkpoint
+                    Lumberjack.info(`Searching local collection for checkpoint.`, getLumberBaseProperties(documentId, tenantId));
+                    checkpoint = await this.localCheckpointCollection.findOne( {documentId, tenantId }).catch((error) => {
```

Comment:
I just wanted to make sure that this is avoiding the table scan and using point read.

Reply:
The collection was built to enable point read once @zhangxin511 completes the database rebuild, but currently it is is not using point read, since the database updates haven't been completed

Reply:
@kekachmar , localChecpointCollection is not conflict with documents collection. change your filter to 
```
{
 _id: value of documentId
documentId, 
tenanntId
}
```

Will greatly reduce RU usage, which is our major pain on global db right now

Please always provide a valid _id, which is `documentId` in your case